src/test/java/org/broadinstitute/consent/http/db/DAOTestHelper.java

```diff
@@ -71,8 +71,25 @@ public class DAOTestHelper extends AbstractTestHelper {
   @SuppressWarnings("rawtypes")
   private static PostgreSQLContainer postgresContainer;
 
-  @BeforeAll
-  public static void startUp() throws Exception {
+   @Override
+  public void testPlanExecutionStarted(TestPlan testPlan) {
+     /* The tests that extend this class make the necessary assumption that the app will be started
+     and backed by a running database.  When we do not need the containers, we are also indicating
+     we do not need the application within this class, hence we can early return and skip resource
+     construction.
+     This became a necessary optimization because of command line generated output that could not
+     be properly handled by a CI/CD process. */
```

Comment:
In general the java terra code uses google code style https://google.github.io/styleguide/javaguide.html#s4.8.6-comments; Personally I would use `//` but `/*` is OK if it's formatted as in the style guide.
```suggestion
     // The tests that extend this class make the necessary assumption that the app will be started
     // and backed by a running database.  When we do not need the containers, we are also indicating
     // we do not need the application within this class, hence we can early return and skip resource
     // construction.
     //
     // This became a necessary optimization because of command line generated output that could not
     // be properly handled by a CI/CD process.
```